src/objects.c

```diff
@@ -2021,19 +2018,19 @@ void for_each_server(PgPool *pool, void (*func)(PgSocket *sk))
 	struct List *item;
 
 	statlist_for_each(item, &pool->idle_server_list)
-		func(container_of(item, PgSocket, head));
+	func(container_of(item, PgSocket, head));
```

Comment:
This looks confusing after the reformatting. Maybe putting braces around the loop body would help?

Reply:
done